.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -203,7 +203,7 @@ jobs:
             if [[ $NUM_TEST_SHARDS -ne 2 ]]; then
               export SHARD_NUMBER=0
             fi
-            if [[ -n $GITHUB_HEAD_REF && "$USE_CUDA" == 1 ]]; then
+            if [[ -n $GITHUB_HEAD_REF && "$BUILD_ENVIRONMENT" == "pytorch-win-vs2019-cuda10-cudnn7-py3" ]]; then
```

Comment:
nit, is it possible to have an option say `run_smoke_tests_only`, and set it to true for `pytorch-win-vs2019-cuda10-cudnn7-py3` in the generation py? That might be less surprising to read why `pytorch-win-vs2019-cuda10-cudnn7-py3` is special

Reply:
That's true--though it would be more like run_smoke_tests_only_on_pull_requests :p

Reply:
I think I will fix this along with other cleanup in a separate PR--would like to unblock Nikita + others using ci/scheduled sooner

Reply:
I'd agree on this point, I'd like to have less special cases just in general and just have them say exactly what they do instead